src/gsi/hybrid_ensemble_isotropic.F90

```diff
@@ -4035,6 +4042,8 @@ subroutine hybens_grid_setup
   logical,allocatable::vector(:)
   real(r_kind) eps,r_e
   real(r_kind) rlon_a(nlon),rlat_a(nlat),rlon_e(nlon),rlat_e(nlat)
+  character(:),allocatable:: fv3_spec_grid_filename
```

Comment:
to conform to the style of your changes in this PR , I suggest to use name as "fv3_ens_spec_grid_filename"